components/org.apache.stratos.common/src/main/java/org/apache/stratos/common/statistics/publisher/ThriftStatisticsPublisher.java

```diff
@@ -61,54 +57,58 @@ public ThriftStatisticsPublisher(StreamDefinition streamDefinition, String thrif
         this.streamDefinition = streamDefinition;
 
         if (isPublisherEnabled()) {
-        	this.enabled = true;
+            this.enabled = true;
             init();
         }
     }
 
     private boolean isPublisherEnabled() {
```

Comment:
We can reduce some lines it seems,
 private boolean isPublisherEnabled() {
        for (ThriftClientInfo thriftClientInfo : thriftClientInfoList) {
            if (thriftClientInfo.isStatsPublisherEnabled()) {
                return true;
            }
        }
        return false;
    }